### PR TITLE
add s3 pushing script

### PR DESCRIPTION
**Pull request by Xe**
connects to heroku/heroku-buildpack-metrics#6

This needs to have a step where the s3 objects have public facing URL's created.

### CODE REVIEW

File: bin/sync-files.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+if [ ${#tools} -ne 0 ]; then
+  echo "The following tools are missing from \$PATH and are required to run this script"
+  echo -e "\t${tools[@]}"
+  echo
+  exit 1
+fi
+
+# assumes we are in bin, find the directory above that
+cwd="$(cd $(dirname $0); cd ..; pwd)"
```
Comment (freeformz) [inline bin/sync-files.sh]: needed?

**Comment by Xe** [comment]
Blocked by https://github.com/heroku/foundation/issues/2375

File: bin/sync-files.sh (hunk 1 above)
Comment (Xe) [inline bin/sync-files.sh]: Probably not.

**Comment by Xe** [comment]
usage:

```console
$ cd /path/to/agentmon
$ make release
$ bash ./bin/sync-files.sh ./agentmon-version.tar.gz
```

File: bin/sync-files.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+if [ ${#tools} -ne 0 ]; then
+  echo "The following tools are missing from \$PATH and are required to run this script"
+  echo -e "\t${tools[@]}"
+  echo
+  exit 1
+fi
+
+${S3CMD} put -P $1 ${BUCKET}
+echo "https://agentmon-releases.s3.amazonaws.com/$1" >> latest
+${S3CMD} rm ${BUCKET}latest
```
Comment (freeformz) [inline bin/sync-files.sh]: Drop this so it's atomic and the file isn't missing for some amount of time. the put below will just replace the existing latest.

Comment (Xe) [inline bin/sync-files.sh]: @freeformz Fixed in 9b7cefd

**Pull request merged**